odd-platform-ui/src/redux/interfaces/datasetStructure.ts

```diff
@@ -64,14 +64,10 @@ export const DatasetStatsLabelMap: Map<
   ['medianValue', 'Median'],
 ]);
 
-export interface UpdateDataSetFieldInternalDescriptionParams {
+export interface UpdateDataSetFieldFormDataParams {
```

Comment:
Don't we have it among interfaces generated from spec?

Reply:
We have close interface named `DatasetFieldUpdateFormData {
   description?: string;
    labelNames?: Array<string>;
}` but in DataSetField interface labels field has type Label[], so i had to create my own.